packages/flutter_tools/pubspec.yaml

```diff
@@ -52,7 +52,7 @@ dependencies:
   http_multi_server: 3.2.1
   convert: 3.1.1
   async: 2.11.0
-  unified_analytics: 5.8.8
+  unified_analytics: 5.8.8+1
```

Comment:
FYI @eliasyishak we're rolled on master

Reply:
Can we apply the cp label to this PR then?

Reply:
No, the bot will just use `git cherry-pick <SHA>`, which will either fail from a merge conflict or just take the checksum from here as is. You will need to follow https://github.com/flutter/flutter/wiki/Updating-dependencies-in-Flutter#to-update-a-single-dependency-for-cherrypicks to update the version but also generate a correct checksum.

Reply:
So does the workflow look like the following for CPs into `beta` that update just one dependency:
1. Checkout the latest beta candidate branch, `flutter-3.22-candidate.0` in this case
2. Create a separate branch to merge into beta branch, `hotfix-unified-analytics-5.8.8+1`
3. Run `flutter update-packages --cherry-pick-package=unified_analytics --cherry-pick-version='5.8.8+1'` and get the updated checksum
4. Open a PR into beta?

Reply:
Everything but the fourth step--your PR should target the candidate branch.

Reply:
oops, i meant to put the candidate branch there, but sounds good, i've opened it up here: https://github.com/flutter/flutter/pull/146948